openhantek/src/glscope.cpp

```diff
@@ -200,10 +200,11 @@ void GlScope::initializeGL() {
           uniform highp vec4 colour;
           void main() { gl_FragColor = colour; }
     )";
-
+    
     const char *vshaderDesktop = R"(
-          #version 150
-          in highp vec3 vertex;
+          #version 120
+          //in highp vec3 vertex;
+          attribute vec3 vertex;
```

Comment:
See https://github.com/vispy/vispy/issues/242
"attribute" is deprecated, "in" and "out" should be used.

Reply:
Deprecated in language 1.5 but not in 1.2 :) and as i've understood openGL even 4.0 still support  old versions of languages.
Yes old  but actually a lot of such old laptops used in labs )
And I don't see the tech limitations do not support  language
But who is a doctor here ) ok closing